### PR TITLE
Automatically set lastmod

### LINKED ISSUE #82: Auto fill date and lostmod in exampleSite
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently date and lastmod are done manually.

**What you expected to happen**:
Hugo has a way to auto fill these it seems.

**Anything else we need to know?**:
https://github.com/gohugoio/hugo/commit/0dbf79c2f8cd5b1a5c91c04a8d677f956b0b8fe8

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
Removes lastmod variable from all content files and sets it's value automatically.

**Which issue this PR fixes**:
fixes #82

### CODE REVIEW

**Comment by lauragift21** [comment]
I was also working on this issue I had a problem displaying the automated date after adding the settings in the config.toml file. Nice to see it fixed :+1:

**Comment by mpourismaiel** [comment]
@lauragift21 Oh I'm sorry I had no idea you were working on it,

File: exampleSite/content/about-split/index.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
 [sidebar]
     title = "Sidebar"
     content = """
-So much information  
-Phone numbers  
-Details  
+So much information
+Phone numbers
+Details
```
Comment (stp-ip) [inline exampleSite/content/about-split/index.md]: The trailing whitespace is usually rendered as new-line in markdown.
Removing it here leads to it being on one line and breaking the sidebar.

Comment (mpourismaiel) [reply exampleSite/content/about-split/index.md]: Fixed in 32628bbc105e8fe69438381bb8f470c43291fcbe. My editor removes trailing whitespace.

**Pull request merged**